docs/source/configuration.rst

```diff
@@ -429,6 +432,15 @@ And here is an example of server-based authentication using SASL::
     auth_password = SopelIsGreat!  # your bot's password
     auth_target = PLAIN            # default sasl mechanism
 
+Example of authentication to a ZNC bouncer::
+
+    [core]
+    auth_method = server           # select server auth
+    # auth_username can't be used with server authentication, so instead
+    # we combine the ZNC username, network name, and password here:
+    auth_password = Sopel/libera:SopelIsGreat!
+    # Configure your ZNC to handle authentication for the real network.
```

Comment:
There is no option left, and this is part of the config. It feels a bit weird to me, like: is there something missing? Do I need to add an option in here? The example makes it look like there should be something here, even tho I think that's not what you intended.

Reply:
Intended as another nudge for "these are not your NickServ credentials, there is no place for those here, go put them in ZNC settings".